packages/boot/src/boot-strapper.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  Context,
+  inject,
+  Constructor,
+  Binding,
+  BindingScope,
+  resolveList,
+} from '@loopback/context';
+import {
+  Booter,
+  BootOptions,
+  BOOTER_PHASES,
+  CoreBindings,
+  Application,
+} from '@loopback/core';
+import {resolve} from 'path';
+import {BootBindings} from './keys';
+
+import * as debugModule from 'debug';
+const debug = debugModule('loopback:boot:bootstrapper');
+
+export class BootStrapper {
+  constructor(
+    @inject(CoreBindings.APPLICATION_INSTANCE) private app: Application,
+  ) {}
```

Comment:
If the `Bootstrapper` is designed to be a singleton within an `app`, let's document it in the comment.

Reply:
I also think there should be a complete statement to that effect in the TSDoc for the class.